src/angular-multi-select-utils.js

```diff
@@ -49,7 +49,9 @@ angular_multi_select_utils.factory('angularMultiSelectUtils', [
 			 * array or undefined.
 			 */
 			if (typeof(str) === 'string') {
-				return str.split(",").map(s => s.replace(/^\s+|\s+$/g, ''));
+				return str.split(",")
+					.map(s => s.replace(/^\s+|\s+$/g, ''))
+					.filter(s => s.length > 0);
```

Comment:
Yep, so clearly my implementation is naive.

In my case, if `preselect = ""` (which is expected if no filters were selected) then the return was `[""]`. My goal was to just get `[]` because `[""]`, for whatever reason (loki?), would cause the first item in the list list to be preselected. 

Perhaps the proper solution is to attempt to balance the array? Inject another `""` or remove the single `""`? If I have time today, I'll see if that resolves my problem.